### PR TITLE
Added localisation for release 2.21

### PR DESCRIPTION
**Pull request by jenskasemets**
What's in this PR:

- Added missing translations and improved some strings in other languages
- Some new strings are missing in some languages, using English placeholders until they get translated

### CODE REVIEW

File: Wire-iOS/Resources/da.lproj/Localizable.strings [hunk 1]
```diff
@@ -485,6 +485,24 @@
 "self.settings.popular_demand.send_button.title" = "Send Knap";
 "self.settings.popular_demand.send_button.footer" = "Deaktiver for at sende via returtasten.";
 
+// Open in
+"self.settings.external_apps.header" = "Open Links In";
```
Comment (mikeger) [inline Wire-iOS/Resources/da.lproj/Localizable.strings]: It's not danish, or?

File: Wire-iOS/Resources/da.lproj/Localizable.strings [hunk 2]
```diff
@@ -688,7 +706,7 @@
 "error.invite.no_email_provider" = "Konfigurer din e-mail klient for at kunne sende invitationer via e-mail";
 "error.invite.no_messaging_provider" = "Konfigurer din SMS for at kunne sende invitationer vis SMS";
 
-"sketchpad.initial_hint" = "Tryk på farve for at ændre penselstørrelse\nTryk og hold inde for at fylde et område\nRyst for at fjerne et billede";
+"sketchpad.initial_hint" = "Tap colors to change brush size";
```
Comment (mikeger) [inline Wire-iOS/Resources/da.lproj/Localizable.strings]: It's not danish, or?

File: Wire-iOS/Resources/da.lproj/Localizable.strings (hunk 1 above)
Comment (mikeger) [inline Wire-iOS/Resources/da.lproj/Localizable.strings]: Ah, I guess it's the fallback then.

Comment (jenskasemets) [inline Wire-iOS/Resources/da.lproj/Localizable.strings]: It's not translated yet in Crowdin, will include translation once done.

**Pull request merged**